### PR TITLE
Make meter modify logic using the H&S approach.

### PR DESCRIPTION
**Pull request by dpoltavets**
Closes: #2298.

---
This change is [Reviewable](https://reviewable.io/reviews/telstra/open-kilda/2299)

### CODE REVIEW

**Comment by dpoltavets** [comment]
___
*[services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/NbWorkerTopology.java, line 147 at r1](https://reviewable.io/reviews/telstra/open-kilda/2299#-LdDF5Hb2bO1l_Lxhq3v:-LdDNh3z6BAv9MaMxdsP:bt2vr1b) ([raw file](https://github.com/telstra/open-kilda/blob/f8679c5913e9bda628057e0a24b98cf5b5db2e47/services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/NbWorkerTopology.java#L147)):*
<details><summary><i>Previously, IlyaMoiseev (Ilya Moiseev) wrote…</i></summary><blockquote>

Think we should increase default timeout in this case, as we need do switch roundtrip  during processing
</blockquote></details>

Fixed.

**Comment by dpoltavets** [comment]
___
*[services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowMeterModifyHubService.java, line 89 at r1](https://reviewable.io/reviews/telstra/open-kilda/2299#-LdDEap0ARs3V2u-Z0vo:-LdDNjTr7xERMx6RRNs6:bt2vr1b) ([raw file](https://github.com/telstra/open-kilda/blob/f8679c5913e9bda628057e0a24b98cf5b5db2e47/services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowMeterModifyHubService.java#L89)):*
<details><summary><i>Previously, IlyaMoiseev (Ilya Moiseev) wrote…</i></summary><blockquote>

Should add `process(fsm)` here, to remove finished FSM from memory
</blockquote></details>

Fixed.

File: services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsService.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2018 Telstra Open Source
+/* Copyright 2010 Telstra Open Source
```
Comment (sergii-iakovenko) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsService.java]: A typo - 2010?

**Comment by rtretyak** [comment]
This is also expected to resolve #2740

**Pull request merged**